src/content/en/2020/seo.md

```diff
@@ -83,183 +84,227 @@ Response Code | All Devices | Mobile | Desktop
+The combination of `Disallow` within `robots.txt` and `noindex` directive in meta robots were found on only 0.03% of pages. While this method offers _belt and suspenders_ redundancy, a page must not be blocked by a `robots.txt` file in order for an on-page `noindex` directive to be effective.
 
-Interestingly, rendering changed the meta-robots tag in 0.16% of pages. While there is no inherent issue with using JavaScript to add a meta robots tag to a page or change its content, SEOs should be judicious in execution. If a page loads with a noindex in the robots meta tag before rendering, [search engines won't run the JavaScript](https://developers.google.com/search/docs/guides/javascript-seo-basics#use-meta-robots-tags-carefully) that changes the tag value or index the page.
+Interestingly, rendering changed the meta robots tag in 0.16% of pages. While there is no inherent issue with using JavaScript to add a meta robots tag to a page or change its content, SEOs should be judicious in execution. If a page loads with a `noindex` directive in the meta robots tag before rendering, [search engines won't run the JavaScript](https://developers.google.com/search/docs/guides/javascript-seo-basics#use-meta-robots-tags-carefully) that changes the tag value or index the page.
 
 #### Canonicalization
 
-Canonical tags -as described by [Google here](https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls)- are used to specify to search engines which is the preferred canonical URL version to index and rank for a page -the one that is considered to be better representative of it- when there are many URLs featuring the same or very similar content. It is important to note that:
+[Canonical tags](https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls), as described by Google, are used to specify to search engines which is the preferred canonical URL version to index and rank for a page—the one that is considered to be better representative of it—when there are many URLs featuring the same or very similar content. It is important to note that:
 
 - The canonical tag configuration is used along with other signals to select the canonical URL of a page; it is not the only one.
-- Although self-referencing canonical tags are sometimes used, these aren’t a requirement.
+- Although self-referencing canonical tags are sometimes used, these aren't a requirement.
 
-[Last year](../2019/seo#canonicalization), it was identified that 48.34% of mobile home pages were using a canonical tag. This year the number of mobile home pages featuring a canonical tag has grown to 53.61%.
+{# TODO(authors): The intro disclaims upfront that the dataset is limited to home pages. Consider generalizing to "mobile/desktop pages" throughout the rest of the chapter. #}
```

Comment:
@aleyda

Reply:
@rviscomi replaced all mentions of "home pages" to "pages", leaving only those where it actually made sense to mention the "home page".